packages/eslint-plugin/eslint.config.js

```diff
@@ -0,0 +1,10 @@
+import workleapPlugin from "@workleap/eslint-plugin";
+
+const config = [
+    ...workleapPlugin.configs.typescriptLibrary.map(conf => ({
```

Comment:
Why does it requires the usage of a map?

Reply:
because the typescriptLibrary config is an array of config objects, and I want the ignore to apply to the objects within this array, and not to all configs. But as I'm thinking through it, the top level `extends` will scope it down, so I'll change this to not map.